packages/frontend-v2/pages/home.tsx

```diff
@@ -252,6 +250,7 @@ const HomePage: NextPage = () => {
             isOverlay={true}
             coReqErr={undefined}
             preReqErr={undefined}
+            isRemove={isRemove}
```

Comment:
This is where the value of `isRemove` is actually used to style the additional course we render